lib/rules/implicit-arrow-linebreak.js

```diff
@@ -161,7 +155,9 @@ module.exports = {
                     ) {
 
                         // If any arrow functions follow, return the necessary parens fixes.
-                        if (sourceCode.getTokenAfter(arrowToken, isArrowToken) && arrowBody.parent.parent.type !== "VariableDeclarator") {
+                        if (sourceCode.getTokenAfter(arrowToken, isArrowToken) &&
```

Comment:
I know this code was here before, but I'm confused about what it's doing. Wouldn't this condition be true if there is any following arrow token anywhere, even if it's in a different part of the file?

Reply:
Good point. With this in mind, the upcoming commit uses `getFirstTokenBetween` to check between the `arrowToken` and last token of the evaluated node to avoid checking different parts of the file.